Source/Private/ASTipsController.mm

```diff
@@ -37,9 +37,9 @@ @implementation ASTipsController
 
 #pragma mark - Singleton
 
-+ (void)load
```

Comment:
While we're on this, can we wrap this initializer in the `AS_ENABLE_TIPS` compiler flag and get back a tiny bit of pre-main time?

Reply:
All of the code in this file is wrapped in `#if AS_ENABLE_TIPS` already, so I think we're good here.

Reply:
👍 Good call. I missed that.